packages/web/src/pages/sign-up-page/components/AccountHeader.tsx

```diff
@@ -30,6 +30,8 @@ type AccountHeaderProps = {
   size?: 'small' | 'large'
   formDisplayName?: string
   formProfileImage?: ImageFieldValue
+  onProfileImageChange?: (value: ImageFieldValue) => void
```

Comment:
Nit: I _think_ I personally prefer just dispatching here instead of prop drilling (but I can easily see the argument for props drilling too)

Reply:
yea I agree, I wasn't getting updates for the selector values in the finish profile page for some reason though so I just went with this